### PR TITLE
Convert +c to |_| from ackbij2 through gchac

### PR DESCRIPTION
**Pull request by jkingdon**
Many of the proofs here can be thought of as lemmas for https://us.metamath.org/mpeuni/gchac.html .

Quite a large number of `+c` theorems can now be removed (and are, in this pull request). The main ones remaining are those downstream of https://us.metamath.org/mpeuni/xpsc.html but that's for a future pull request.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -99209,18 +99209,6 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
     wa domtr ) ACEBDERZAABFZGHZUAABIZGHAUCGHTUAJEAUAKUBABCDLABMAUAJNOABCDPAUAUC
     SQ $.
 
-  $( Cartesian product dominates disjoint union for sets with cardinality
-     greater than 1.  Similar to Proposition 10.36 of [TakeutiZaring] p. 93.
-     (Contributed by Mario Carneiro, 18-May-2015.)  Use ~ djuxpdom instead.
-     (New usage is discouraged.) $)
-  cdaxpdom $p |- ( ( 1o ~< A /\ 1o ~< B ) -> ( A +c B ) ~<_ ( A X. B ) ) $=
```
Comment (wlammen) [inline set.mm]: The description of this commit should be "Remove..."

File: changes-set.txt [hunk 2]
```diff
@@ -26,6 +26,43 @@ make a github issue.)
...
+17-Sep-23 cdafn     ---         |_| is not a function
+17-Sep-23 cdaenun   djuenun     Changes from +c notation to |_|
+17-Sep-23 cdadom1   djudom1     Changes from +c notation to |_|
+17-Sep-23 cdacomen  djucomen    Changes from +c notation to |_|
+17-Sep-23 cdadom2   djudom2     Changes from +c notation to |_|
+17-Sep-23 pwxpndom2cda pwxpndom2 Changes from +c notation to |_|
+17-Sep-23 xp2cda    xp2dju      Changes from +c notation to |_|
+17-Sep-23 uncdadom  undjudom    Changes from +c notation to |_|
+17-Sep-23 mapcdaen  mapdjuen    Changes from +c notation to |_|
+17-Sep-23 cdadom3   djudoml     Changes from +c notation to |_|
```
Comment (wlammen) [inline changes-set.txt:40]: Both cdadom3 and cdadom1 are replaced with djudom1.  Is this ok?

Comment (jkingdon) [reply changes-set.txt:40]: The replacement for cdadom3 is `djudoml` (with the letter l) and the replacement for cdadom1 is `djudom1` (with the digit 1).

File: set.mm (hunk 1 above)
Comment (jkingdon) [reply set.mm]: Thanks; fixed.

File: changes-set.txt (hunk 2 above)
Comment (wlammen) [reply changes-set.txt:40]: Thanks for the explanation.

**Review by wlammen — APPROVED** [review]
The work is done very systematically, so I am convinced this is ok.

**Comment by jkingdon** [comment]
> The work is done very systematically, so I am convinced this is ok.

Well I'm not infallible, so if anyone sees anything later, speak up.

Merging now for all the usual reasons including reducing the chance for merge conflicts.

**Pull request merged**